### PR TITLE
feat: Use hex-prefixed string, and dash instead of underscore for subgraph querying

### PR DESCRIPTION
**Pull request by ukstv**
Apparently, subgraph queries use dash as a delimiter for token id

### CODE REVIEW

File: src/__tests__/index.test.ts [hunk 1]
```diff
@@ -9,10 +9,10 @@ import { Caip10Link } from '@ceramicnetwork/stream-caip10-link'
 import * as u8a from 'uint8arrays'
 import fetchMock from 'jest-fetch-mock'
 import { ethers } from 'ethers'
-import { NftDidVectorBuilder, NftDidVector } from '../testUtils/NftDidVector'
+import { NftDidVectorBuilder, NftDidVector } from './nft-did-vector'
 import ganache from 'ganache-core'
 
-const ERC721_QUERY_URL = 'https://api.thegraph.com/subgraphs/name/wighawag/eip721-subgraph'
+const ERC721_QUERY_URL = 'https://api.thegraph.com/subgraphs/name/touchain/erc721track'
```
Comment (oed) [inline src/__tests__/index.test.ts:15]: Hm, why was this changed? does this track all erc712 tokens?

Comment (ukstv) [reply src/__tests__/index.test.ts:15]: "does this track all erc712 tokens" as far as I can tell, yes.

whighawag's subgraph uses underscore for token id delimiter (contract-id pair), which seems to be incompatible with every other 721/1155 subgraphs I tried using (including `amxx/eip1155-subgraph` included originally).

**Review by oed — APPROVED** [review]
